pkg/controller/issueapi/issue.go

```diff
@@ -308,14 +308,13 @@ func (c *Controller) HandleIssue() http.Handler {
 				}
 
 				logger.Errorw("failed to send sms", "error", err)
-				stats.Record(ctx, mCodeIssueErrors.M(1))
-				stats.RecordWithTags(ctx, []tag.Mutator{observability.APIResultError("NOT_OK")}, mSMSRequest.M(1))
+				stats.RecordWithTags(ctx, []tag.Mutator{observability.ResultError("NOT_OK")}, mSMSRequest.M(1))
 				c.h.RenderJSON(w, http.StatusInternalServerError, api.Errorf("failed to send sms"))
 				blame = observability.BlameServer
-				result = observability.APIResultError("FAILED_TO_SEND_SMS")
+				result = observability.ResultError("FAILED_TO_SEND_SMS")
 				return
 			}
-			stats.RecordWithTags(ctx, []tag.Mutator{observability.APIResultOK()}, mSMSRequest.M(1))
+			stats.RecordWithTags(ctx, []tag.Mutator{observability.ResultOK()}, mSMSRequest.M(1))
```

Comment:
Is it intended that there's no "item" here?

Reply:
Yeah this is a metric specifically for SMS requests.